build.sbt

```diff
@@ -43,15 +44,20 @@ val dep = {
     "org.scala-sbt.ipcsocket" % "ipcsocket" % "1.0.0",
     "org.bouncycastle" % "bcprov-jdk15on" % "1.59",
     "com.typesafe.scala-logging" %% "scala-logging" % "3.9.0",
-    "org.typelevel" %% "mouse" % "0.18",
+    "org.typelevel" %% "mouse" % "0.23",
```

Comment:
Maybe we could start adding new Dependencies following the approach we use in the other project. (like i started doing it here: https://github.com/input-output-hk/mantis/pull/668/files#diff-fdc3abdfd754eeb24090dbd90aeec2ceR77) WDYT?

Reply:
Yes, we should do that. I will do that in separate PR